data/json/npcs/TALK_COMMON_OTHER.json

```diff
@@ -13,6 +13,30 @@
       { "text": "Well, bye.", "topic": "TALK_DONE" }
     ]
   },
+  {
+    "id": "TALK_FRIEND",
+    "type": "talk_topic",
+    "responses": [
+      {
+        "text": "Relax and chat with me for a while...",
+        "topic": "TALK_FRIEND_CHAT",
+        "condition": { "not": { "npc_has_effect": "asked_to_socialize" } }
+      }
+    ]
+  },
+  {
+    "id": "TALK_FRIEND_CHAT",
+    "type": "talk_topic",
+    "dynamic_line": "Ok, so, how's it going?",
```

Comment:
Consider adding a snippet here so the NPC will make random small talk.

Reply:
#27638